diagnostic.md

```diff
@@ -62,7 +62,7 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
+result = 0, -5 The value of the result is false because it returned the second value.
```

Comment:
We want the actual returned value here which is 5, not a boolean (true or false) value.